From a49974ba3f835973f5f28447363dced300b8a560 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Sun, 12 Nov 2017 19:04:55 +0100
Subject: [PATCH 1054/1089] ARM: dts: suni-a83t-tbs-a711: Enable I2S0 for
 communication with AC100

This will allow for sound playback after AC100 audio codec is
implemented.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index f22da26557f2..bc08c3c281c1 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -207,6 +207,12 @@
 	};
 };
 
+&dai {
+	status = "okay";
+	pinctrl-names = "default";
+	pinctrl-0 = <&i2s0_pins>;
+};
+
 &de {
 	status = "okay";
 };
-- 
2.19.2

